components/cart.jsx

```diff
@@ -7,17 +7,18 @@ import { formatPrice } from 'lib/utils/text';
 import { getCheckoutPayload } from 'lib/utils/checkout';
 import { CreateMyCheckoutSession } from 'lib/queries';
 import { useCart } from 'lib/cart';
-import getStripe from 'lib/utils/stripe';
 import { ProductPrice, ProductQuantitySelect } from './products';
 import { isTypeSystemExtensionNode } from 'graphql';
-import { client } from 'lib/utils/shopify';
+import { shopifyClient } from 'lib/utils/shopify';
+import { useApolloClient } from '@apollo/client';
 
 export const CartIcon = () => {
   const {
     items,
     isCartReady,
     actions: { toggleCart }
   } = useCart();
+  const client = useApolloClient();
```

Comment:
Why is this here? Your linter should have flagged it as unused — you might need to check that it's working properly.